### PR TITLE
Breed/node install start

### PR DESCRIPTION
**Pull request by BRReed**
everything else I just merged the repos, but for this one I wanted to separate getting node and the npm install / start portion of the node app deploy. major change is using a variable for the lsof check on line 11 of the install-start.sh

I tested this and it worked on jammy.

### CODE REVIEW

File: nodejs-management/install-start/ubuntu/install-start.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+if ( npm ls )
+then
+  echo "Dependencies have been installed"
+else
+  echo "Installing dependencies"
+  npm install
+fi
+
+if ( lsof -i:$1 )
```
Comment (stephenlauck) [inline nodejs-management/install-start/ubuntu/install-start.sh]: wtf is that?

Comment (BRReed) [reply nodejs-management/install-start/ubuntu/install-start.sh]: The variable will be the port the app is deployed on, `lsof -i` lists internet address files and `:<port>` returns what matches. so if there is no match we need to start the service, if there is a match the service is running (or something is already running on the port and you fucked up somewhere else)

shortened: `lsof -i<match>` returns a list of files whose address fully or partially matches whats in the brackets

Comment (stephenlauck) [reply nodejs-management/install-start/ubuntu/install-start.sh]: If you're doing a real deploy you would probably be using a systemd service unit file. Then you would check if there are running services like:

`systemctl is-active myapp.service`

You could also just curl the URL:PORT and see if you get a 200. No one is going to understand what LSOF is doing, so there are tradeoffs to readability/understanding if you go that direction. That's def cool though.

Comment (BRReed) [reply nodejs-management/install-start/ubuntu/install-start.sh]: ok I changed it to a curl command and now testing if it returns 200. 

Let me know if you have a better way. I stole the curl command from stackoverflow then modified to use in the test.

**Review by stephenlauck — APPROVED** [review]

**Pull request merged**